packages/at_chops/lib/src/key/asymmetric_key_pair.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:typed_data';
+
+/// Represents a key pair for asymmetric public-private key encryption/decryption
+abstract class AsymmetricKeyPair<Pub extends AtPublicKey,
```

Comment:
AsymmetricKeyPairs can now be flavored with covariants of AtPublicKey and AtPrivateKey. Non breaking change, as they still conform to the same interface.